<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WKSPUN05DR37313" name="TestCase_004" tests="1" failures="0" timestamp="28 Jul 2015 05:35:40 GMT" time="0.093" errors="0">
  <testcase name="@BeforeTest checkCaseToRun" time="0.0" classname="com.qcim.SuiteTwo.Verify_LoansTabRecords">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.qcim.utility.ExcelUtils.retrieveToRunFlag(ExcelUtils.java:287)
	at com.qcim.utility.Utils.checkToRunUtility(Utils.java:30)
	at com.qcim.SuiteTwo.Verify_LoansTabRecords.checkCaseToRun(Verify_LoansTabRecords.java:53)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeTest checkCaseToRun -->
  <testcase name="@AfterMethod reporterDataResults" time="0.0" classname="com.qcim.SuiteTwo.Verify_LoansTabRecords">
    <skipped/>
  </testcase> <!-- @AfterMethod reporterDataResults -->
  <testcase name="@AfterTest afterMethod" time="0.0" classname="com.qcim.SuiteTwo.Verify_LoansTabRecords">
    <skipped/>
  </testcase> <!-- @AfterTest afterMethod -->
  <testcase name="TestCase_004" time="0.0" classname="com.qcim.SuiteTwo.Verify_LoansTabRecords">
    <skipped/>
  </testcase> <!-- TestCase_004 -->
</testsuite> <!-- TestCase_004 -->
